ethers-signers/Cargo.toml

```diff
@@ -16,7 +16,7 @@ rustdoc-args = ["--cfg", "docsrs"]
 
 [dependencies]
 ethers-core = { version = "^1.0.0", path = "../ethers-core", features = ["eip712"] }
-thiserror = { version = "1.0.38", default-features = false }
+thiserror = { version = "1.0.36", default-features = false }
```

Comment:
this doesn't actually change the locked version. 
So I don't understand what exactly this fixes.

Reply:
I temporarily used `version = "=1.0.36"` to change the lock. Once the lock file was updated, I removed the `=` sign since it struck me as quite restrictive.